### ISSUE TITLE
Extra second prior to Dec 13, 1901

### ISSUE THREAD

**Issue by elidchan** [sdispater/pendulum#152]
```
>>> Pendulum(1901, 12, 13, 0, 0, 9, 555555, 'US/Central')
<Pendulum [1901-12-13T00:00:10.555555-06:00]>
```

Notice the 9 seconds became 10 seconds.

Dates after December 13, 1901 don't have this issue:
```
>>> Pendulum(1901, 12, 14, 0, 0, 9, 555555, 'US/Central')
<Pendulum [1901-12-14T00:00:09.555555-06:00]>

```
This issue seems to be dependent upon the microseconds term:
```
>>> Pendulum(1901, 12, 13, 0, 0, 9, 0, 'US/Central')
<Pendulum [1901-12-13T00:00:09-06:00]>

```
It also seems to be dependent upon the time zone:
```
>>> Pendulum(1901, 12, 13, 0, 0, 9, 555555, 'UTC')
<Pendulum [1901-12-13T00:00:09.555555+00:00]>

```
December 13, 1901 seems to correspond with the minimum value of a 32-bit signed integer:
https://www.drupal.org/node/1708296

**Comment by sdispater** [comment]
Thanks for reporting this.

This is a weird bug indeed. I am not sure where it comes from.

I will investigate and get back to you.

**Comment by sdispater** [comment]
This is fixed in commit c0d37bd7823ae67367a5662e1da29585a2b7ef96.

It will land in the next bugfix release.

**Comment by elidchan** [comment]
Thanks for fixing this - much appreciated!
